src/Settings/Comments.php

```diff
@@ -1,46 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace P4\MasterTheme\Settings;
-
-use P4\MasterTheme\Features\GdprCheckbox;
-use P4\MasterTheme\Loader;
-
-/**
- * Settings related to Posts comments.
- */
-class Comments
-{
-    public const OPTIONS_KEY = 'planet4_comments';
```

Comment:
The old key for saving the data was `planet4_comments`, we need to use the same key to use existing settings value.

Reply:
If I'm not mistaken, this was only the Settings section key, but the checkbox itself that was used for the GDPR setting was `gdpr_checkbox`? Anyway, now that I've seen [Flo's PR](https://github.com/greenpeace/planet4-master-theme/pull/2090) for the reading time setting I'm gonna refactor mine a bit 😬

Reply:
In old implementation, if you print `get_option('planet4_comments');` you will get the `gdpr_checkbox` value.
I will revisit it after refactoring 🙂

Reply:
Ah 🤔 I got confused because [we use the GdprCheckbox component](https://github.com/greenpeace/planet4-master-theme/blob/7c150a6b563139eb38b073bdad4f8d53167beef4/single.php#L107) in the frontend and we never use `get_option('planet4_comments');`... I'll let you know once it's refactored, sorry about that!

Reply:
no problem 👍